src/components/video-box/VideoBox.tsx

```diff
@@ -1,20 +1,26 @@
 import { FC } from 'react'
 import Box from '@mui/material/Box'
-
 import titleBar from '~/assets/img/guest-home-page/title-bar.png'
 import { styles } from '~/components/video-box/VideoBox.styles'
+import VideoPlayer from '../video-player/VideoPlayer'
 
 interface VideoBoxProps {
   video: string
+  videoPreview: boolean
+  videoMock: string
 }
 
-const VideoBox: FC<VideoBoxProps> = ({ video }) => {
+const VideoBox: FC<VideoBoxProps> = ({ video, videoPreview, videoMock }) => {
+  const videoBlockContent = videoPreview ? (
+    <Box alt='Video' component='img' src={videoMock} sx={styles.video} />
```

Comment:
use `ComponentEnum`
For `component` and also for other code properties where necessary

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/6d1ff246-0412-4308-96f6-1500271edb30)
fixed